### PR TITLE
feat: cart management

### PR DESCRIPTION
**Pull request by aatishpandey**
Cart Management
image

### CODE REVIEW

File: src/contexts/cartContext.js [hunk 1]
```diff
@@ -0,0 +1,31 @@
+import { createContext, useContext, useReducer } from "react";
+import { cartReducer } from "../utils/CartReducer";
+import { useAxios } from "../utils/useAxios";
+import { useState, useEffect } from "react";
+
+const CartContext = createContext();
+
+const CartProvider = ({ children }) => {
+    const [cartState, cartDispatch] = useReducer(cartReducer, {
```
Comment (rahulcs754) [inline src/contexts/cartContext.js:9]: here you can set state name userState like that bcoz you are not storing only cart details. here you store wishlist with cart.

**Review by rahulcs754 — APPROVED** [review]
The code looks good and clean.
Just Small suggestion

**Review by SimranjitKaurMaan — APPROVED** [review]
Address the other suggestions. Good to go.

File: src/contexts/cartContext.js (hunk 1 above)
Comment (aatishpandey) [reply src/contexts/cartContext.js:9]: thanks for pointing out.

**Pull request merged**